### DISCUSSION TITLE
MasterTemp Heaters?

### DISCUSSION THREAD

**Discussion by nrobfd** [tagyoureit/nodejs-poolController#700] [Nixie] [answered]
Was planning to buy a Intellicenter, but thinking I'm going this route instead. Going to be replacing all my equipment this spring and want to add automation at the pad. I have some pi & automation experience. One thing that I wanted to check was if the MasterTemp can be controller via RS485 instead of just the fireman's switch?

My current goals are as follows:
Control Pump Speed
Control SWG
Control Heater
Control Lights
Monitor Temp
Monitor Automatic Cover

In the future I'd like to add monitoring / dosing of PH, monitoring of chlorine level, control of valves. 

It seems Nixie is more than capable of all this.

**Comment by rstrouse** [comment, marked as answer]
Yes the MasterTemp protocol is integrated into Nixie.  She speaks and listens in MasterTemp parseltongue.
